addons/website/views/website_templates.xml

```diff
@@ -296,6 +302,38 @@
     </xpath>
 </template>
 
+<!-- Default template Mega Menu -->
+<template id="default_mega_menu" name="Mega Menu">
+    <div class="s_mega_menu m-2 p-2">
+        <div class="s_mega_menu_content container text-center">
+            <div class="row s_mega_menu_columns">
+                <t t-call="website.default_mega_menu_col"/>
+                <t t-call="website.default_mega_menu_col"/>
+                <t t-call="website.default_mega_menu_col"/>
+            </div>
+        </div>
+    </div>
+</template>
+
+<template id="default_mega_menu_col" name="Mega Menu Demo Column">
```

Comment:
Don't need this, just do a foreach in your `default_mega_menu` template

Reply:
for each what ? This is a template I use into the default mega menu template that we have by default when we create a mega menu

Reply:
`t-foreach="3"`, as explained in the doc

Reply:
Yes I understand that, I just don't really understand the difference in our case

Reply:
- No three identical t-call in the main template (duplicated code)
- No extra template in the database just for mega menu default column
- Less code for the same stuff without any loss of feature

Reply:
Seems like you bugged after doing point 1 of my list here. The whole point of the `foreach` is to remove the extra template. Could you move the default col content instead of the t-call please..? Thank you.